### DISCUSSION TITLE
precision parameter gets ignored in merge?

### DISCUSSION THREAD

**Discussion by Ixiodor** [corteva/rioxarray#240] [Q&A]
I'm working on porting merge from rasterio to rioxarray. Used to have a problem of gaps between patches(doing mosaic) in rasterio solved using "precision=50". Tryed to do the same thing in rioxarray and i get the result as precision was not specified. Tryed to lower too, like 10 where I used to have gaps in rasterio and seems there are no changes.

**Comment by snowman2** [comment]
There isn't a great solution for this as rioxarray doesn't support decimated reads. It rounds to the nearest grid cell when doing window selection. This is likely the reason the precision doesn't have much impact.
